macros/core/PGbasicmacros.pl

```diff
@@ -455,11 +455,9 @@ sub NAMED_ANS_BOX {
 		TeX        => qq!\\vskip $height in \\hrulefill\\quad !,
 		Latex2HTML =>
 			qq!\\begin{rawhtml}<TEXTAREA NAME="$name" id="$name" aria-label="$label" ROWS="$row" COLS="$col"
-		              WRAP="VIRTUAL">$answer_value</TEXTAREA>\\end{rawhtml}!,
-		HTML => qq!<TEXTAREA NAME="$name" id="$name" ROWS="$row" COLS="$col"
-		        WRAP="VIRTUAL">$answer_value</TEXTAREA>
-		        <INPUT TYPE=HIDDEN  NAME="previous_$name" VALUE = "$answer_value">
-		        !,
+				>$answer_value</TEXTAREA>\\end{rawhtml}!,
+		HTML => qq!<TEXTAREA NAME="$name" id="$name" ROWS="$row" COLS="$col">$answer_value</TEXTAREA>!
```

Comment:
Should this TEXTAREA have the aria-label on it?

Reply:
Yeah, it should.   In fact, a mistake that traces back to be0e7139c (a @goehle commit) is apparent here.  Looking at the code above, aria labels were added by mistake to the Latex2HTML code but not to the actual HTML code.  I will move that.  The Latex2HTML should not have that at all (and will soon be deleted anyway).